www-apps/hugo/files/0.92.2_link-to-webp-and-sass.patch

```diff
@@ -0,0 +1,38 @@
+From b1c99224dc0d318109832531c98ec24ef4c0020e Mon Sep 17 00:00:00 2001
```

Comment:
Please start patch names with `${P}-` for consistency with the common ebuild style.